tests/integration/primary_pool_tests.cpp

```diff
@@ -560,3 +568,98 @@ TYPED_TEST(PrimaryPoolTest, heuristic_75_percent)
   ASSERT_NO_THROW({ alloc.deallocate(a[0]); });  // 100% releasable
   ASSERT_EQ(dynamic_pool->getBlocksInPool(), 1); // Collapse happened
 }
+
+template <typename PoolTuple>
+class PrimaryPoolTimingsTest : public ::testing::Test {
+ public:
+  using Pool = typename camp::at<PoolTuple, camp::num<0>>::type;
+  using ResourceType = typename camp::at<PoolTuple, camp::num<1>>::type;
+
+  void SetUp() override
+  {
+    m_resource_name = std::string(tag_to_string<ResourceType>::value);
+    m_allocator = build_allocator<Pool, false>("pool_", 100);
+    m_allocator_no_coalesce = build_allocator<Pool, false>("no_coalesce_pool", 0);
+  }
+
+  void TearDown() override
+  {
+    delete m_allocator;
+    delete m_allocator_no_coalesce;
+  }
+
+  void run_test(umpire::Allocator* alloc)
+  {
+    int max_time{0};
+
+    if (std::is_same<Pool, umpire::strategy::DynamicPoolList>::value) {
+      max_time = 800;
```

Comment:
We need to review these maximum times.  I made them about 30% greater than the best times I was seeing on rzhasgpu.  Not sure if this will be sufficient across all of the platforms we build on.

Reply:
Took this out.